neurodocker/interfaces/afni.py

```diff
@@ -161,3 +186,62 @@ def install_binaries(self):
+        """Return Dockerfile instructions to download and build AFNI from
+        source."""
+        pkgs = self._get_source_dependencies()
+
+        cmd = ("{install}"
+               "".format(**manage_pkgs[self.pkg_manager]).format(pkgs=pkgs))
+
+        if self.pkg_manager == "yum":
+            cmd += ("\n&& yum install -y -q R R-devel")
+
+        if self.pkg_manager == "apt":
+            # libxp was removed after ubuntu trusty.
+            deb_url = ('http://mirrors.kernel.org/debian/pool/main/libx/'
+                       'libxp/libxp6_1.0.2-2_amd64.deb')
+            cmd += ('\n&& echo "Install libxp (not in all ubuntu/debian repositories)"'
+                    "\n&& apt-get install -yq --no-install-recommends libxp6"
+                    '\n|| /bin/bash -c "'
+                    '\n   curl --retry 5 -o /tmp/libxp6.deb -sSL {}'
+                    '\n   && dpkg -i /tmp/libxp6.deb && rm -f /tmp/libxp6.deb"'
+                    ''.format(deb_url))
+
+            cmd += ('\n&& ln -s /usr/lib/x86_64-linux-gnu/libXp.so.6.2.0 /usr/lib/x86_64-linux-gnu/libXp.so'
+                    '\n&& ln -s /usr/lib/x86_64-linux-gnu/libXmu.so.6.2.0 /usr/lib/x86_64-linux-gnu/libXmu.so')
+
+        cmd += ("\n&& {clean}"
+                '\n&& echo "Downloading AFNI ..."'
+                "\n&& cd /opt"
+                "\n&& git clone https://github.com/afni/afni.git"
+                "\n&& cd afni"
+                "\n&& git checkout {}"
+                "\n&& cd src"
+                "\n&& cp Makefile.linux_openmp_64 Makefile"
+                "\n&& perl -p -i -e 's/^LGIFTI.*/LGIFTI    = -lexpat/' Makefile"
+                "\n&& perl -p -i -e 's/^USE_LOCAL_X_TREE/#USE_LOCAL_X_TREE/' Makefile"
+                "\n&& perl -p -i -e 's/XLIBS = \$\(XROOT\)\/lib64\/libXm.a -lXt/XLIBS = \$\(XROOT\)\/lib64\/libXm.a \$\(XROOT\)\/lib\/x86_64-linux-gnu\/libXm.a -lXt/' Makefile"
+                "\n&& perl -p -i -e 's/^# XLIBS =/XLIBS =/' Makefile"
+                "\n&& perl -p -i -e 's/^CCOLD.*/CCOLD  = \$\(CC\)/' Makefile"
+                "\n&& perl -p -i -e 's/(^LFLAGS.*)/$1 -L\/usr\/lib\/x86_64-linux-gnu/' Makefile"
+                "\n&& perl -p -i -e 's/(^PLFLAGS.*)/$1 -L\/usr\/lib -L\/usr\/lib\/x86_64-linux-gnu/' Makefile"
+                "\n&& perl -p -i -e 's/-lXpm -lXext/-lXpm -lfontconfig -lXext/' Makefile"
+                "\n&& perl -p -i -e 's/(^SUMA_INCLUDE_PATH.*)/$1 -I\/usr\/lib\/x86_64-linux-gnu\/glib-2.0\/include/' Makefile"
+                "\n&& make INSTALLDIR=/opt/afni vastness"
+                "".format(self.version, **manage_pkgs[self.pkg_manager]))
+
+        if self.install_r:
+            cmd += ("\n&& /opt/afni/rPkgsInstall -pkgs ALL"
+                    "\n&& rm -rf /tmp/*")
+
+        cmd = indent("RUN", cmd)
+
+        env_plugin_path = "AFNI_PLUGINPATH=/opt/afni"
+        env_plugin_path = indent("ENV", env_plugin_path)
+
+        env_cmd = "PATH=/opt/afni:$PATH"
+        env_cmd = indent("ENV", env_cmd)
```

Comment:
Can you merge `ENV AFNI_PLUGINPATH=/opt/afni` and `ENV PATH=/opt/afni:$PATH` into one `ENV` layer?

Reply:
Good idea. Done.